### DISCUSSION TITLE
Where are the embeddings/indexings/vectors stored on my local machine?

### DISCUSSION THREAD

**Discussion by Hunter-Bobeck** [logancyang/obsidian-copilot#558] [Q&A]
I want to make sure that my data is in safe places. And I want to exclude it from Obsidian Sync.

**Comment by logancyang** [comment]
It's in your Obsidian app's local indexedDB, not included in Obsidian Sync. Someone asked for support for Sync before though. It seems it's better to have both options available.

**Comment by le-patenteux** [comment]
i would like to know more... what exact files are created when we index our vault? Can these vector files be transferred to another AI service, such as Open-WebUI ?

**Reply by logancyang** [reply]
This logic is going to change significantly and I'll have this info in the documentation. As for transferring to another service, probably no. Copilot will have unique AI agent functionalities and the index will reflect that.
